README.md

```diff
@@ -554,3 +563,19 @@ An alias to the first message in the messages collection.
 get(user, 'validations.message')
 get(user, 'validations.attrs.username.message')
 ```
+
+## Templating Example ##
+Accessing validation information in your templates is really simple. Oh and don't worry, we know the path is pretty long! There are plans to condense that in the roadmap ahead.
+
+```handlebars
+<form>
+  {{input value=model.username placeholder="Username"}}
+  {{#if model.validations.attrs.username.isInvalid}}
```

Comment:
I really like the idea of the combined `v-get` helper. the whole `model.validations.attrs.field.prop` is long to type.